hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -831,7 +828,7 @@ public static HoodieTableFileSystemView getFileSystemView(HoodieTableMetaClient
    * @param datasetMetaClient                   - Data table meta client
    * @param isMetaIndexColumnStatsForAllColumns - Is column stats indexing enabled for all columns
    */
-  private static List<String> getLatestColumns(HoodieTableMetaClient datasetMetaClient, boolean isMetaIndexColumnStatsForAllColumns) {
+  private static List<String> getColumnsToIndex(HoodieTableMetaClient datasetMetaClient, boolean isMetaIndexColumnStatsForAllColumns) {
```

Comment:
a comment about L 834. I feel we can't directly take in RecordKeyFieldProp as is. may not work for all key gens. 
may be we have to split with "," and then set the columns to index. 
Can you think if there are any other places where we have this dependency and check if we have done the right thing

Reply:
sure. do we have a tracking jira?

Reply:
I did not create a separate jira.. This is already being tracked in https://issues.apache.org/jira/browse/HUDI-3411